src/Netscript/RamCostGenerator.ts

```diff
@@ -226,6 +226,18 @@ const singularity = {
   getUnlockedAchievements: SF4Cost(RamCostConstants.SingularityFn3),
 } as const;
 
+const cloud = {
+  getCloudServerLimit: RamCostConstants.GetPurchasedServerLimit,
```

Comment:
These constants are misnamed now, and having them in extra constants just makes the RAM costs harder to follow. I'd take the opportunity to inline these.